conf/messages

```diff
@@ -10,6 +12,9 @@ fileChecksResults.description.plural=Your folder ''{0}'' containing {1} items ha
 
 transferComplete.error=There has been an error, contact us.
 
+judgmentTransferAgreement.warning=We cannot publish the court judgment without this: Name(s) of judge(s), name of parties, court, judgment date, neutral citation, case reference(s).
```

Comment:
Not sure what the designs say, but don't think the word `this` is correct: _... the court judgment without the: Name(s) ..._